README.md

```diff
@@ -74,20 +56,37 @@ cy status -o json | jq '.[] | select( .status != "Success")'
 ### Create config repository using credential named "Git Config"
 
 ```
-GIT_CRED=$(cy --org $ORG credential list -o json | jq '.[] | select( .name == ""Git Config") | .id')
+GIT_CRED=$(cy --org $ORG credential list -o json | jq '.[] | select( .name == ""Git Config") | .canonical')
 cy --org myorg  config-repository create --branch master --cred $GIT_CRED --name "lab-config" --default --url "git@github.com:org/repo.git"
 ```
 
 ### Invite members with "Admin" role
 
 ```
-ADMIN_ROLE=$(cy --org $ORG  roles list -o json | jq '.[] | select( .name | contains("Admin")) | .id')
-cy  --org myorg  members invite --role-id $ADMIN_ROLE --email foo@email.com
+ADMIN_ROLE=$(cy --org $ORG  roles list -o json | jq '.[] | select( .name | contains("Admin")) | .canonical')
+cy  --org myorg  members invite --role $ADMIN_ROLE --email foo@email.com
 ```
 
 :construction:
 <!-- This is where we could add some useful examples: create a user, etc. -->
 
+## Build from the sources
+
+You can clone the repository and build from sources (we use `Go Modules` (1.13+)):
```

Comment:
To build the CLI from source code, first ensure that have a working Go environment with version 1.13 or greater installed and the `make` command available.
After that, you can clone the repository yourself, build using `make build` and move the built binary where you want it to be (eg. in `/usr/local/bin` for a global install):